tutorials/rnn/ptb/ptb_word_lm.py

```diff
@@ -60,6 +60,7 @@
 from __future__ import print_function
 
 import time
+from functools import partial
```

Comment:
I'd prefer using lambda config, is_training: tf.contrib..XXX()

Reply:
I believe that config is not a good idea to begin with. As I was trying to grok the code, having config passed as argument from method to method building the graph had one simply wonder what exactly the methods need from config. IOW this:

    def _build_rnn_graph_lstm(self, inputs, rnn_mode, hidden_size, num_layers,
                                           keep_prob, batch_size, is_training):

is much more easy to follow than:
 
      def _build_rnn_graph_lstm(self, inputs, config, is_training):

plus it shows that our model depends on the batch_size (not good ?). `is_training` calls for a subclass.

I would suggest going all the way to argparse dropping the configs - maybe keeping them as "presets"